clients/java/src/main/java/io/unitycatalog/client/api/ModelApi.java

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Once this is regenerated, will it be renamed to RegisteredModelApi? One side-effect of making sure we're consistent with the Databricks OpenAPI spec is that these names should mostly become consistent

Reply:
I needed to rename the tag at the top.  Fixing.